config/bootstrap.rb

```diff
@@ -32,3 +32,10 @@ def perform
     add_transaction_tracer :perform, :category => :task
   end
 end
+
+require 'routemaster/mixins/log'
+include Routemaster::Mixins::Log
```

Comment:
Hmm that was the way chosen in `bin/worker` so I wanted to stick with that by thinking we wouldn't want everything to be global space of course but some things such as logging helpers might be useful. 

For lazy path, I guess we need to either monkey patch or extend `Logger` to sneak our check in? Extending sounds cleaner to me. What do you think?